### PR TITLE
perf: Remove IE11 from browserslist and babel targets

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [x] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

babel 插件的相关改动：https://github.com/ant-design/antd-tools/pull/269

应该能优化不少体积。这里是 `antd/lib/alert/index.js` 文件的变动：https://github.com/afc163/test-git/pull/4/files

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   Remove IE and other legacy browsers from browserslist to reduce 10KB in bundle size.  |
| 🇨🇳 Chinese | 在构建流程中去掉对 IE 等旧版本浏览器的支持以减少包体积 10KB。  |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
不知道为啥构建挂了。。

**Comment by li-jia-nan** [comment]
conflict

**Comment by yoyo837** [comment]
> 有一个问题，我们能直接用 class 么：https://caniuse.com/?search=classes

个人觉得干脆`es`目录直接输出`module js`, 那自然很多都是`class`的了

**Comment by yoyo837** [comment]
rc-*的 lib 引用全部换掉

**Comment by afc163** [comment]
不知道为啥 CI 会挂。

图片

**Review by li-jia-nan — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -158,7 +156,7 @@
     "throttle-debounce": "^5.0.0"
   },
   "devDependencies": {
-    "@ant-design/tools": "^16.1.0-alpha.2",
+    "@ant-design/tools": "17.0.0-alpha.5",
```
Comment (afc163) [inline package.json:159]:
```suggestion
    "@ant-design/tools": "^17.0.0-alpha.5",
```

**Comment by zombieJ** [comment]
诡异，本地不报错。CI 报错

**Comment by zombieJ** [comment]
重试又好了，只能当是抽风了。 。 。

**Review by zombieJ — APPROVED** [review]

**Comment by afc163** [comment]
不知道为啥会抽风，合并了再观察观察。

**Pull request merged**